src/_zkapauthorizer/_plugin.py

```diff
@@ -140,7 +140,8 @@ def get_storage_server(
             registry=registry,
         )
         storage_server = ZKAPAuthorizerStorageServer(
-            anonymous_storage_server,
+            # unwrap the Foolscap layer, we'll do it ourselves.
+            anonymous_storage_server._server,
```

Comment:
Based on running the PrivateStorageio system tests, this is incorrect, as `anonymous_storage_server` is already a `StorageServer`, not a `FoolscapStorageServer`.

Reply:
Thanks for this catch.  I'm surprised this is what Tahoe is doing now ... but since it is easier to deal with than what I expected, I guess I won't complain. :slightly_smiling_face: